src/Components/CategoryChips/CategoryChips.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react'
+import { useSearchParams } from 'react-router-dom';
+import { useVideo } from '../../Context/VideoContext/VideoContext';
+import './CategoryChips.css'
+
+function CategoryChips() {
+
+  const { videostate, selectcategory, videodispatch } = useVideo()
```

Comment:
Just a suggestion: use camelCase as much as possible

Reply:
Sure i will update that